ferrocene/doc/evaluation-report/src/rustc/tool-analysis.rst

```diff
@@ -276,22 +276,22 @@ Moreover, this analysis shows us that the likelihood of detecting these
 potential errors is very low. Therefore, the tool error detection class is
 **TD3**.
 
-Using clause 11.4.5.4 in part 8 of the [|iso_ref|] standard, we can conclude that in
-the worst case the Tool Classification Level is **TCL3** and therefore we choose
-the following qualification methods:
+Using clause 11.4.5.4 in part 8 of the [|iso_ref|] standard, we can conclude
+that in the worst case the Tool Classification Level is **TCL3/ASIL D** and
```

Comment:
Why did you change the classifcation level to be `TCL3/ASIL D`? The TCL is just `TCL3`.

Reply:
Because before the situation was like following

- In the "Tool Evaluation Results" section we said `"the worst case the Tool Classification Level is **TCL3**"`
- In the "IEC 61508 Tool Classification" section we said `"to conform with automotive [|iso_ref|] TCL 3/ASIL D level"`

To me it does not make sense that we talk about ISO in the IEC section. Therefore I removed that mention from there. But since in the ISO section (Before "Tool Evaluation Result", now "ISO 26262 Tool Classification") we did not mention the ASIL D, which was mentioned in the IEC section, I added it there.

Reply:
Oh I see!

I think saying "tool classification level" of "TCL3/ASIL D" is wrong, as ASIL D is not the tool classification level. I'd change it to be just TCL3.